app/views/spree/admin/payments/show.html.haml

```diff
@@ -10,7 +10,8 @@
 - content_for :page_actions do
   %li= button_link_to Spree.t(:back_to_payments_list), spree.admin_order_payments_url(@order), icon: 'icon-arrow-left'
 
-= render partial: "spree/admin/payments/source_views/#{@payment.payment_method.method_type}", locals: { payment: @payment.source.is_a?(Spree::Payment) ? @payment.source : @payment }
+-unless @payment.payment_method.nil?
```

Comment:
this is a minor thing but there should I space in between the dash and the word. I find it weird that Rubocop doesn't complain.

Reply:
Now that I look at it again, isn't a bit more straightforward to make it positive like `- if @payment.payment_method.present?`? I feel the `unless` + `nil` a bit of brain gymnastics.

Reply:
I think it'd be easier to comprehend that way. You're right.